### PR TITLE
boring documentation cleanup

### PR DESCRIPTION
**Pull request by drunkirishcoder**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,11 @@
-# nb2
\ No newline at end of file
+# nb2
+
+An opinionated framework for network function development. Written in Rust, inspired by [NetBricks](http://netbricks.io/) and built on Intel's [Data Plane Development Kit](https://www.dpdk.org/).
```
Comment (zeeshanlakhani) [inline README.md]: please do not use the word opinionated here, sounds uncool yo.

File: README.md [hunk 2]
```diff
@@ -1 +1,11 @@
...
+
+# Overview
+
+The project is currently undergoing heavy development. The goal is to offer an ergonomic framework for network function development that traditionally has high barriers of entry for engineers. We want a tool to efficiently manipulate network packets that's type safe, memory safe and thread safe.
```
Comment (zeeshanlakhani) [inline README.md]: We are building a tool to efficiently manipulate network packets while being type-safe, memory-safe, and thread-safe.

File: README.md [hunk 3]
```diff
@@ -1 +1,11 @@
...
+
+# Quickstart
+
+We are working on better documentation and a "Getting started" guide. For now, if you are interested in trying out the pre-release bits, head over to our [docker images](https://github.com/williamofockham/utils) repo and follow the directions to create a local development environment with DPDK and nb2. Then dig around the examples to see for yourself how easy it is to write a program that can filter, parse and create network packets.
```
Comment (zeeshanlakhani) [inline README.md]: Remove Then, replace with `Afterward, dig around the examples and see for yourself` 

Also, `filter, parse, and...`

File: core/src/packets/ip/v6/mod.rs [hunk 4]
```diff
@@ -419,7 +405,7 @@ pub const IPV6_PACKET: [u8; 78] = [
     0x20, 0x01, 0x0d, 0xb8, 0x85, 0xa3, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01,
     // dst addr
     0x20, 0x01, 0x0d, 0xb8, 0x85, 0xa3, 0x00, 0x00, 0x00, 0x00, 0x8a, 0x2e, 0x03, 0x70, 0x73, 0x34,
-    // ** TCP header
+// TCP header
```
Comment (zeeshanlakhani) [inline core/src/packets/ip/v6/mod.rs]: aligning in these parts seem weird no?

File: core/src/packets/ip/v6/srh.rs [hunk 5]
```diff
@@ -444,14 +405,51 @@ impl<E: Ipv6Packet> Ipv6Packet for SegmentRouting<E> {
...
+    }
+}
+
+impl Header for SegmentRoutingHeader {}
+
 #[cfg(any(test, feature = "testils"))]
 #[rustfmt::skip]
 pub const SRH_PACKET: [u8; 170] = [
-    // ** ethernet header
+// ethernet header
```
Comment (zeeshanlakhani) [inline core/src/packets/ip/v6/srh.rs]: same as above.

File: core/src/packets/ip/v6/mod.rs (hunk 4 above)
Comment (drunkirishcoder) [reply core/src/packets/ip/v6/mod.rs]: on purpose, marks where each header section starts and standout from the regular comments with no indentation. before I was use `// **`, didn't really standout and it's weird af.

Comment (zeeshanlakhani) [reply core/src/packets/ip/v6/mod.rs]: yeah, still reads a bit weird, but ok.

**Comment by ari23** [comment]
pretty cool stuff guys!! I tested ping6d just to get my toes wet and its working flawlessly!

**Comment by drunkirishcoder** [comment]
@ari23 thank you, glad you found it helpful.

**Pull request merged**